content/docs/admin/mail-config.md

```diff
@@ -0,0 +1,92 @@
++++
+title = "Mail Configuration"
+description = "Configuring Bookstack to send mail"
+date = "2017-08-08"
+type = "admin-docs"
+slug = "mail-config"
++++
+
+This document has been written by referencing the file at `config/mail.php`.
+
+Bookstack supports the following mechanisms for sending mail:
+
+1. SMTP;
+2. mail (php sendmail() function);
+3. sendmail (Linux sendmail);
+4. Mailgun (Rackspace Mailgun);
+5. Mandrill;
+6. Amazon SES ('ses'); or
+7. Log
+
```

Comment:
To keep things as simple as possible for now, I only really want to include 1, 2 & 3.

The services for 4,5,6 can all be used via SMTP anyway to it doesn't fully exclude them by removing them from the docs. Also, They ideally need an additional dependencies installed in the main BookStack code to function reliably.

Also remove 7 to keep things simple.